package/scripts/presto_client.py

```diff
@@ -80,7 +80,8 @@ def ensure_nodes_are_up(client):
         raise RuntimeError('Presto server failed to start within \
 {0} seconds.'.format(RETRY_TIMEOUT))
 
-
+# This class was copied more or less verbatim from
+# https://github.com/prestodb/presto-admin/blob/master/prestoadmin/prestoclient.py
```

Comment:
Yea it's possible to extract the Python presto client. However, it would be a non trivial amount of work. As I was writing this code I realized that anyway we should release the presto Python client as a separate project for data scientists and such. So at some point we'll do it but right now I'm focused on completing this integration.